lib/active_fedora/orders/association.rb

```diff
@@ -82,6 +82,12 @@ def delete_at(loc)
       target.delete_at(loc)
     end
 
+    # Delete all occurences of the specified target
+    # @param obj object to delete
+    def delete(obj)
```

Comment:
If we delete the proxy, won't that mess up the members (unordered)?  I need an operation that is just like delete_at(), but deletes ever occurrence in the ordered list.